core/src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java

```diff
@@ -522,42 +546,27 @@ else if (!entrez.matches("[0-9]+")) {
                                 }
                             }                            
                         } else {
-                            boolean hasMicroRNA = false;
-                            int otherCase = 0;
-                            for (CanonicalGene gene : genes) {
-                                if (gene.isMicroRNA() || rppaProfile) { // for micro rna or protein data, duplicate the data
-                                    if (gene.isMicroRNA()) {
-                                        hasMicroRNA = true;
-                                    }
+                            if (rppaProfile) { // for protein data, duplicate the data
+                                for (CanonicalGene gene : genes) {
                                     boolean result = storeGeneticAlterations(values, daoGeneticAlteration, gene, geneSymbol);
                                     if (result == true) {
                                         recordStored = true;
                                         nrExtraRecords++;
                                     }
                                 }
-                                else {
-                                    otherCase++;
-                                }
                             }
                             if (recordStored) {
                                 //skip one, to avoid double counting:
                                 nrExtraRecords--;
                             }
```

Comment:
Instead of `if (recordStored) {} if (!recordStored) {}` we can just do an `if (recordStored) {} else {}`.